File: EXPORT/old/v0.3.0/v0.3.0/scripts/helix_plot.m
% helix_plot.m - 3D helix visualization
t = 0:pi/50:10*pi;
x = sin(t); 
y = cos(t); 
z = t;

% Generate data file
fid = fopen('helix_data.csv', 'w');
fprintf(fid, '# t, x, y, z\n');
for i = 1:length(t)
    fprintf(fid, '%.6f, %.6f, %.6f, %.6f\n', t(i), x(i), y(i), z(i));
end
fclose(fid);

disp('Helix data exported to helix_data.csv');
disp(['Points: ' num2str(length(t))]);

File: EXPORT/old/v0.3.0/examples/scripts/helix_plot.c
// helix_plot.c - 3D helix data generator
#include <stdio.h>
#include <math.h>

int main() {
    const double pi = 3.14159265358979323846;
    const int n = 100;
    
    printf("# 3D Helix Plot Data\n");
    printf("# t, x, y, z\n");
    
    for (int i = 0; i < n; i++) {
        double t = (i / (double)n) * 10 * pi;
        double x = sin(t);
        double y = cos(t);
        double z = t;
        printf("%.6f, %.6f, %.6f, %.6f\n", t, x, y, z);
    }
    
    printf("\n# Export complete: helix_plot.csv\n");
    return 0;
}
